crates/aptos/src/move_tool/mod.rs

```diff
@@ -83,6 +83,10 @@ impl CliCommand<Vec<String>> for CompilePackage {
 pub struct TestPackage {
     #[clap(flatten)]
     move_options: MovePackageDir,
+
+    /// A filter string to determine which unit tests to run
+    #[clap(name = "filter", short = 'f', long = "filter")]
```

Comment:
We're skipping short names since they can get overloaded between commands.

you can just put `#[clap(long)]` here since it's named the same.